.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+version: 2.1
+
+orbs:
+    browser-tools: circleci/browser-tools@0.1.0
+
+executors:
+  snovault-executor:
+    docker:
+      - image: circleci/buildpack-deps:trusty
+    working_directory: ~/snovault
+    environment:
+      - BASH_ENV: /home/circleci/.bashrc
+      - ES_JAVA_OPTS: -Xms2g -Xmx2g
+      - JAVA_HOME: /usr/lib/jvm/java-11-openjdk-amd64
+
+commands:
+  buildout:
+    description: A buildout process based on snovault-executor
+    steps:
+      - checkout
+      # Do not use caching for now to avoid potential conflicts/bugs
+      - run:
+          name: Add apt repositories
+          command: |
+            sudo apt-get install apt-transport-https software-properties-common
+            wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+            echo "deb https://artifacts.elastic.co/packages/5.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-6.x.list
+            curl -sSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
+            curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+            sudo add-apt-repository -y ppa:openjdk-r/ppa
+            sudo apt-add-repository -y ppa:brightbox/ruby-ng
+            sudo apt-get update
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt-get install -y \
+              bsdtar \
+              elasticsearch \
+              graphviz \
+              nodejs \
+              openjdk-11-jdk \
+              postgresql-9.3 \
+              ruby2.3 \
+              ruby2.3-dev
+            sudo chown -R circleci /etc/elasticsearch
+            sed -i '1s;^;export PATH=/usr/share/elasticsearch/bin:/usr/lib/postgresql/9.3/bin:$PATH\n;' $BASH_ENV
```

Comment:
we're updating to postgres 11 at some point in the near future.  It has a different installation and some of the directories change.  Is it possible to make some of these strings into variables?
```
PATH=%(ES_BIN)s:%(PG_BIN)s:$PATH
```
or maybe for the node url above
```
echo "%(NODE_URL)s stable main"
```
Just curious how we may accomplish this.  We have similar lines in all the cloud configs so it might be nice to have one version var file...

Reply:
It's definitely possible, the first method that comes to mind is environment variables. (see https://circleci.com/docs/2.0/env-vars/ ). You could read them in from a file for sure as well

Reply:
cool.  We're not ready for this but lets try to keep it in mind. It would be nice to eventually integrate this with cloud configs.

Reply:
nvm, you already did it. :)

Reply:
Saw your comment too late haha. ES, Postgres, and Node version numbers are now environment variables and not hard coded, so they'll be easier to change later